File: P13_2403/p13_2403.py
def firstNonRepeating(arr, n):
      
    for i in range(n):
        j = 0
        while(j < n):
            if (i != j and arr[i] == arr[j]):
                break
            j += 1
        if (j == n):
            return arr[i]
      
    return -1
      

arr = [ 8, 1, 8, 6, 7, 1 ]
n = len(arr)
print(firstNonRepeating(arr, n))

File: P13_2403/p13_2403.cpp
#include <stdio.h>

  
int firstNonRepeating(int arr[], int n)
{
    for (int i = 0; i < n; i++) {
        int j;
        for (j = 0; j < n; j++)
            if (i != j && arr[i] == arr[j])
                break;
        if (j == n)
            return arr[i];
    }
    return -1;
}
  

int main()
{
    int arr[] = { 8, 1, 8, 6, 7, 1 };
    int n = sizeof(arr) / sizeof(arr[0]);
    printf("%d",firstNonRepeating(arr, n)); 
    return 0;
}